end_to_end_tests/golden-record/my_test_api_client/api/tests/defaults_tests_defaults_post.py

```diff
@@ -122,12 +112,14 @@ def _get_kwargs(
         "union_prop": json_union_prop,
         "union_prop_with_ref": json_union_prop_with_ref,
         "enum_prop": json_enum_prop,
-        "nullable_model_prop": json_nullable_model_prop,
-        "nullable_required_model_prop": json_nullable_required_model_prop,
     }
     if not isinstance(json_model_prop, Unset):
         params.update(json_model_prop)
     params.update(json_required_model_prop)
+    if not isinstance(json_nullable_model_prop, Unset) and json_nullable_model_prop is not None:
+        params.update(json_nullable_model_prop)
+    if json_nullable_required_model_prop is not None:
+        params.update(json_nullable_required_model_prop)
```

Comment:
I think this is the right call, it matches the intended default of `style=query,explode=true`, but it is a breaking change we'll have to remember to note.